sdk/python/feast/infra/offline_stores/offline_utils.py

```diff
@@ -130,6 +133,25 @@ def get_feature_view_query_context(
         event_timestamp_column = feature_view.input.event_timestamp_column
         created_timestamp_column = feature_view.input.created_timestamp_column
 
+        min_event_timestamp = None
+        if feature_view.ttl:
+            min_event_timestamp = entity_df_timestamp_range[0] - feature_view.ttl
+            if min_event_timestamp.tzinfo is not None:
```

Comment:
this probably can be moved into some helper function (actually `_to_naive_utc` already exists)